app/views/funding/services/_documents.html.haml

```diff
@@ -18,41 +18,34 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
-#custom-toolbar
-  .row.tabs{role: 'tabpanel'}
-    #document_tabs{data: {selected: "loi"}}
-      .col-lg-6
-        %button.btn.btn-primary.btn-block.document_nav#documents_loi
-          %span.pull-left
-            %span.glyphicon.glyphicon-file
-            %span
-              = t(:funding)[:download][:loi]
-          %span.glyphicon.glyphicon-refresh.pull-right
-      .col-lg-6
-        %button.btn.btn-default.btn-block.document_nav#documents_application
-          %span.pull-left
-            %span.glyphicon.glyphicon-file
-            %span
-              = t(:funding)[:download][:application]
-          %span.glyphicon.glyphicon-refresh.pull-right{style: "display: none;"}
+.row.mb-3#documents_tabs{data: {selected: "loi"}}
+  .col-lg-6
+    %button.btn.btn-primary.btn-lg.btn-block.document_nav#documents_loi
```

Comment:
Adding a `<` character at the end of this line (no space between) will remove the extra whitespace between the icon and text